### DISCUSSION TITLE
Issue with creating databases with a . in the name

### DISCUSSION THREAD

**Discussion by bcarney66** [Sequel-Ace/Sequel-Ace#1676] [General] [answered]
Thank you for creating Sequel Ace, I love the software!

I may have found a bug.  If I go to the Database pulldown menu and select Add Database, I can create one with a . in the name - wp_mydb.new for example.

If I use the query window and use CREATE DATABASE wp_mydb.new;  I get an error message:

"You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '.old' at line 1"

**Comment by Jason-Morcos** [comment, marked as answer]
You probably need to wrap the name of the database in quotes in the manually query if you're putting special characters in its name :)

If you go to the console in Sequel Ace, you can see the command that Ace ran when you created the database via the GUI. When you're using the query window, Ace isn't mutating the queries for you - it's simply running whatever you type and displaying the result
